testdata/baselines/reference/submoduleAccepted/compiler/checkJsdocTypeTagOnExportAssignment1.errors.txt.diff

```diff
@@ -1,25 +1,11 @@
 --- old.checkJsdocTypeTagOnExportAssignment1.errors.txt
 +++ new.checkJsdocTypeTagOnExportAssignment1.errors.txt
 @@= skipped -0, +0 lines =@@
--a.js(8,18): error TS2353: Object literal may only specify known properties, and 'c' does not exist in type 'Foo'.
--
--
+ a.js(8,18): error TS2353: Object literal may only specify known properties, and 'c' does not exist in type 'Foo'.
+
+
 -==== checkJsdocTypeTagOnExportAssignment1.js (0 errors) ====
```

Comment:
all that's left is a diff in the test framework I think

Reply:
https://github.com/microsoft/typescript-go/pull/1014 will fix this.